client/python/lib/ovmsclient/tfs_compat/http/serving_client.py

```diff
@@ -19,6 +19,12 @@
 
 class HttpClient(ServingClient):
 
+    def __init__(self, address, port, client_key=None, server_cert=None):
+        self.address = address
+        self.port = port
+        self.client_key = client_key
+        self.server_cert = server_cert
```

Comment:
Why are we storing client_key and server_cert in HttpClient?

Reply:
We plan to use requests library for sending HTTP requests. As far as I know, we need to pass paths to the final method, so they are needed in HttpClient so it can call POST or GET methods with SSL.